assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -45,19 +45,23 @@ const Space = styled.div`
   flex: 0 0 10px;
 `;
 
-function PaddingControls({ selectedElements, pushUpdateForObject }) {
+function PaddingControls({
+  selectedElements,
+  pushUpdateForObject,
+  pushUpdate,
+}) {
   const padding = useCommonObjectValue(
     selectedElements,
     'padding',
     DEFAULT_PADDING
   );
-
-  const [lockPaddingRatio, setLockPaddingRatio] = useState(true);
+  const rawLockPadding = getCommonValue(selectedElements, 'lockPadding');
```

Comment:
Why shouldn't the `lockPadding` be instead `padding.locked`? It's slightly different from aspect ratio, but that's not critical.

Reply:
`lockPadding` and removed instead used `locked` inside `padding` and update the tests based on that. And update the `locked` for multiple elements when they don't have common lock ratio.